app-frontend/src/app/components/scenes/sceneFilterPane/sceneFilterPane.controller.js

```diff
@@ -75,214 +147,192 @@ export default class FilterPaneController {
-        this.datefilterPreset = 'None';
-        this.hasDatetimeFilter = false;
+    initFilterSlideOptions() {
+        this.filterOptions = {
+            cloudCover: {
+                minModel: cloudCoverRange.min,
+                maxModel: cloudCoverRange.max,
+                options: {
+                    floor: cloudCoverRange.min,
+                    ceil: cloudCoverRange.max,
+                    minRange: 0,
+                    showTicks: 10,
+                    showTicksValues: true,
+                    step: 10,
+                    pushRange: true,
+                    draggableRange: true,
+                    onEnd: (id, minModel, maxModel) => {
+                        this.onFilterUpdate({
+                            minCloudCover: minModel !== cloudCoverRange.min ? minModel : null,
+                            maxCloudCover: maxModel !== cloudCoverRange.max ? maxModel : null
+                        });
+                    }
+                }
+            },
+            sunElevation: {
+                minModel: sunElevationRange.min,
+                maxModel: sunElevationRange.max,
+                options: {
+                    floor: sunElevationRange.min,
+                    ceil: sunElevationRange.max,
+                    minRange: 0,
+                    showTicks: 30,
+                    showTicksValues: true,
+                    step: 10,
+                    pushRange: true,
+                    draggableRange: true,
+                    onEnd: (id, minModel, maxModel) => {
+                        this.onFilterUpdate({
+                            minSunElevation: minModel !== sunElevationRange.min ? minModel : null,
+                            maxSunElevation: maxModel !== sunElevationRange.max ? maxModel : null
+                        });
+                    }
+                }
+            },
+            sunAzimuth: {
+                minModel: sunAzimuthRange.min,
+                maxModel: sunAzimuthRange.max,
+                options: {
+                    floor: sunAzimuthRange.min,
+                    ceil: sunAzimuthRange.max,
+                    minRange: 0,
+                    showTicks: 60,
+                    showTicksValues: true,
+                    step: 10,
+                    pushRange: true,
+                    draggableRange: true,
+                    onEnd: (id, minModel, maxModel) => {
+                        this.onFilterUpdate({
+                            minSunAzimuth: minModel !== sunAzimuthRange.min ? minModel : null,
+                            maxSunAzimuth: maxModel !== sunAzimuthRange.max ? maxModel : null
+                        });
+                    }
+                }
+            }
+        };
     }
 
-    setDateRange(start, end, preset) {
-        if (_.isEmpty({start}) || _.isEmpty(end)) {
-            this.clearDateFilter();
+    setFilters(filters) {
+        if (filters && filters.maxCloudCover) {
```

Comment:
It would be a bit cleaner to just have 
```javascript
if (!filters) {
  this.resetAllFilters();
  return;
}
```
and make `resetAllFilters()` also set default values for all the `filterOptions`
instead of checking for `filter` in every if statement